.meteor/packages

```diff
@@ -16,9 +16,9 @@ standard-minifiers      # JS/CSS minifiers run for production mode
 es5-shim                # ECMAScript 5 compatibility for older browsers.
 ecmascript              # Enable ECMAScript2015+ syntax in app code
 
-autopublish             # Publish all data to the clients (for prototyping)
```

Comment:
Removing autopublish, will make us think about published data from the start. In order for data to be accessible from the client, we will have to "publish" it at the server.

You can think of data "publications" as REST endpoints if you like. You ask endpoints to bring you data (by subscribing to them) and so they do.